src/Timeline/components/Group.js

```diff
@@ -16,15 +16,26 @@ const formatTime = (timestamp) =>
 
 const last = (array) => array[array.length - 1];
 
-const mapDateAndTime = (event, idx, events) => {
+const mapDateAndTime = (event, idx, events, prevEvents) => {
   const mappedEvent = { ...event, time: formatTime(event.timestamp) };
   const isMajor = idx === events.length - 1;
+  const date = formatDate(event.timestamp, isMajor);
+
+  // Get previous date
+  const prevDate = !!prevEvents
+    ? last(prevEvents.map(mapDateAndTime))?.date
+    : null;
```

Comment:
Could we replace this with 
`const prevDate = !!prevEvents
    && last(prevEvents.map(mapDateAndTime))?.date;`?

Reply:
Haha yeah no worries, will approve

Reply:
Or will try approve it